ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Insights.kt

```diff
@@ -34,25 +36,36 @@ import react.useEffect
 
 @RFunction
 fun RBuilder.insights(components: List<AppComponent>) {
-    div(classes = "row mb-3") {
```

Comment:
Let's actually reorder the charts:
1. Component type graphs
2. File type graphs
3. Resource type graphs

That way we don't hide something on the top and on the bottom (probably better UX-wise).

Reply:
Already reordered and simplified!